kibana-details/impersonation.md

```diff
@@ -0,0 +1,137 @@
+# Impersonation 
+
+After describing what [the impersonation is](../kibana.md#impersonate), it's high time to see how does ROR support it and who and when could be interested in using this feature. Let's start with the latter one.
+
+## Use cases
+
+The impersonation feature is prepared for a ROR admin. It's not rather useful for users (at least not directly). We can point out two the most obvious use cases when the admin could take advantage of the feature:
```

Comment:
-> The impersonation feature is intended for ROR administrators, rather than users.